src/ResourceManager/Insights/Commands.Insights/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -31,6 +32,11 @@ namespace Microsoft.Azure.Commands.Insights.Diagnostics
     [Cmdlet(VerbsCommon.Set, "AzureRmDiagnosticSetting"), OutputType(typeof(PSServiceDiagnosticSettings))]
```

Comment:
Can we just leave the name as Set-DiagnsoticSettings? We are improving the api to support multiple diagnostic settings. After that we plan to retire this commandlet and have only Add-AzureRmDiagnosticSettings. I don't think we should change the name a command that is soon to be deprecated.